packages/client/src/styles/globals.css

```
@@ -1512,12 +1517,104 @@
+.mari-quick-preset-editor {
+  --marinara-editor-bg: transparent;
+  --marinara-editor-surface-bg: var(--marinara-chat-chrome-panel-bg);
+  --marinara-editor-surface-bg-hover: var(--marinara-chat-chrome-highlight-bg-hover);
+  --marinara-editor-control-bg: var(--marinara-chat-chrome-button-bg);
+  --marinara-editor-control-bg-hover: var(--marinara-chat-chrome-button-bg-hover);
+  --marinara-editor-border: var(--marinara-chat-chrome-button-border);
+  --marinara-editor-border-strong: var(--marinara-chat-chrome-button-border-hover);
+  --marinara-editor-divider: var(--marinara-chat-chrome-panel-divider);
+  --marinara-editor-text: var(--marinara-chat-chrome-panel-text);
+  --marinara-editor-muted: var(--marinara-chat-chrome-panel-muted);
+  background: transparent;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)